components/views/landing/Landing.styles.tsx

```diff
@@ -108,24 +117,8 @@ export const StyledLanding = styled.section`
             padding: var(--content-padding);
             padding-top: 3rem;
         }
-    }
 
-    @media only screen and (max-width: 1350px) {
-        .overlay {
-            display: flex;
-            .overlay-text {
-                display: flex;
-                align-content: center;
-                flex-direction: column;
-                align-items: flex-start;
-                height: 100%;
-                width: 100%;
-                justify-content: center;
-                padding-left: 1.6rem;
-            }
-        }
-        .services {
-            background-color: var(--Secondary-accent-color);
+        @media only screen and (max-width: 1350px) {
```

Comment:
So this is one of those times where you have the exact same media query, max width of 1350 in the same file. Write it once, where both classes can be accessed by it, and call each class within it and apply the styles there. The opposite of our last PR comment exchange 🤣